### PR TITLE
add directions and images for downloading ssl certificate from browser

### PR DESCRIPTION
**Pull request by cldougl**
@chriddyp

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -376,10 +376,24 @@ <h5 id="MATLAB-SSL-Error">
...
+Alternatively, you can download the certificate from your browser by:
+<br>
+1. Clicking on the padlock icon next to the URL
+<img style="max-width: 100%" src="https://images.plot.ly/plotly-documentation/images/SSLErr1.jpg"/>
+2. Select 'More Information'
+3. Select 'View Certificate'
+4. Select 'Details'
+5. Select Export and save as a .crt file
+<img style="max-width: 100%" src="https://images.plot.ly/plotly-documentation/images/SSLErr2.jpg"/>
+<img style="max-width: 100%" src="https://images.plot.ly/plotly-documentation/images/SSLErr1.jpg"/>
```
Comment (chriddyp) [inline index.html]: should this be https://images.plot.ly/plotly-documentation/images/SSLErr3.jpg ?

Comment (chriddyp) [inline index.html]: could we save it as `certs.crt` *or* change https://github.com/plotly/dashboards.ly/pull/10/files#diff-eacf331f0ffc35d4b482f1d15a887d3bL380 to match `DSTRootCAX3.crt`? https://github.com/plotly/dashboards.ly/pull/10/files#diff-eacf331f0ffc35d4b482f1d15a887d3bL391 depends on the filename `cert.crt` so I could see that being a point of confusion

Comment (cldougl) [inline index.html]: :bulb:  I'll save it as certs.crt

**Comment by cldougl** [comment]
@chriddyp I edited the screenshot as well to show saving as `certs.crt` so that should be more clear now.

**Pull request merged**